src/PockyBot.NET.Persistence/Repositories/PegRepository.cs

```diff
@@ -0,0 +1,21 @@
+using PockyBot.NET.Persistence.Models;
+
+namespace PockyBot.NET.Persistence.Repositories
+{
+    public class PegRepository : IPegRepository
+    {
+        private readonly DatabaseContext _context;
+
+        public PegRepository(DatabaseContext context)
+        {
+            _context = context;
+        }
+
+        public Peg CreatePeg(Peg peg)
+        {
+            var added = _context.Add(peg);
+            _context.SaveChanges();
```

Comment:
I think I've seen most async code around, but why not these repositories?

Reply:
No reason, I'll make it async